src/mechanics/elements/ElementBase.h

```diff
@@ -71,8 +71,9 @@ class ElementBase
 
 public:
     //! @brief constructor
-    //! @param rInterpolationType ... interpolation type
-    ElementBase(const InterpolationType& rInterpolationType);
+    //! @param interpolationType ... interpolation type
+    //! @param integrationType ... integration type
+    ElementBase(const InterpolationType& interpolationType, const IntegrationTypeBase& integrationType);
```

Comment:
Should the ElementBase class contain the integration type as well as the constitutive models? In the current setup, this is perfectly fine, since for the "next generation" pde class this will change anyway (hopefully).